src/App.tsx

```diff
@@ -20,12 +30,16 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                value={query}
+                onChange={(eve) => {
```

Comment:
Use a more descriptive name for the variable `eve`? It would greatly enhance the clarity and maintainability of the code.
```suggestion
                onChange={(event) => {
```